install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -24,13 +24,23 @@ data:
     # Deprecated: mixer is using EDS
```

Comment:
this is the deprecation i am speaking about.

Reply:
lost one of my comments.  Isn't mixerCheckServer and mixerReportServer deprecated?  If its meaning has changed, perhaps we should change the variables as well so folks reading the code will not expect old behavior when experiencing new behavior.